pkg/utils/utils.go

```diff
@@ -67,28 +69,18 @@ func (errs combinedError) Error() string {
 }
 
 // ErrorGroup contains a set of non-nil errors
-type ErrorGroup []error
+type ErrorGroup errs.Group
```

Comment:
Makes sense.. I'll avoid this in the future. The reason I did this was because I encountered some old code that was using `ErrGroup` and thought it would be nice to make it consistent everywhere that was still using it.

Reply:
I agree, but the fix for that would be to start replacing `utils.ErrorGroup` package by package and finally remove `utils.ErrorGroup` as the last step.

Reply:
yea, I sort of saw this as an intermediary step to keep the implementations the same.

Reply:
should I roll it back or is it fine to keep it?

Reply:
If it doesn't cause test failures, then it's fine.

Reply:
:+1: